bmaptools/BmapHelpers.py

```diff
@@ -83,3 +94,72 @@ def program_is_available(name):
             return True
 
     return False
+
+
+def get_mount_point(filepath):
+    """
+    Return the mount point the file object file resides on.
+    """
+    f = os.path.realpath(filepath)
+    while f != os.path.sep:
+        if os.path.ismount(f): return f
```

Comment:
Please, keep the return statement on a separate line. It may look unimportant, and it is indeed not a big deal, but I am a fan of consistency and I find code much easier to read when it is uniform. So let's mimic the existing code (and this is a general rule of thumb for contributions - mimic existing code, if willing to improve something, improve globally).